orc8r/cloud/deploy/orc8r_deployer/docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM python:3.9-slim-buster
-
+ARG ENV=prod
```

Comment:
This is a dangerous pattern in general. The default should be prod -- and we only do things differently if `ENV=test`

Reply:
Actually the "prod" name doesn't make sense here. I should have named this differently. But the default is indeed the docker contained without binaries used for test. Will clean it up in a follow up PR.